catalog/dags/common/storage/db_columns.py

```diff
@@ -79,3 +81,10 @@
     col.STANDARDIZED_POPULARITY,
     col.AUDIO_SET_FOREIGN_IDENTIFIER,
 ]
+
+DB_COLUMNS_BY_MEDIA_TYPE = {AUDIO: AUDIO_TABLE_COLUMNS, IMAGE: IMAGE_TABLE_COLUMNS}
+
+
+def setup_db_columns_for_media_type(func: callable) -> callable:
+    """Provide media-type-specific DB columns as a kwarg to the decorated function."""
+    return setup_kwargs_for_media_type(DB_COLUMNS_BY_MEDIA_TYPE, "db_columns")(func)
```

Comment:
FWIW, these implementations can be simplified a bit to:

```suggestion
setup_db_columns_for_media_type = setup_kwargs_for_media_type(DB_COLUMNS_BY_MEDIA_TYPE, "db_columns")
"""Provide media-type-specific DB columns as a kwarg to the decorated function."""
```

I'm not sure if that results in a worse docstring experience in the editor, though.

Reply:
Totally! I didn't like the way the docstrings looked, but I actually have no idea what the "ideal" solution is here, that's obviously personal preference. I do think these merit the docstring either way, though.